### PR TITLE
Add initial amp-story examples

### PR DESCRIPTION
**Pull request by newmuis**

### CODE REVIEW

File: examples/amp-story/helloworld.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <style amp-custom>
+      body {
+        font-family: 'Roboto', sans-serif;
+      }
+      h1 {
+        font-size: 2.875em;
+        font-weight: normal;
+        line-height: 1.174;
+        text-transform: uppercase;
+      }
```
Comment (sebastianbenz) [inline examples/amp-story/helloworld.html]: same as above, add:

```
amp-story-page {
  background: white;
}
```

File: examples/amp-story/animations-presets.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      border-radius: 80px;
+    }
+    </style>
+</head>
+<body>
+  <amp-story standalone>
+    <amp-story-page id="fade-in-page">
+      <amp-story-grid-layer template="vertical">
+        <h1>fade-in</h1>
+        <div class="circle"
```
Comment (sebastianbenz) [inline examples/amp-story/animations-presets.html]: Fade-in doesn't seem to work - do animations work for title pages?

File: examples/amp-story/animations-presets.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+      <amp-story-grid-layer template="vertical">
+        <h1>rotate-in-left</h1>
+        <div class="square"
+            animate-in="rotate-in-left">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="rotate-in-right-page">
+      <amp-story-grid-layer template="vertical">
```
Comment (sebastianbenz) [inline examples/amp-story/animations-presets.html]: this fails with an exception:

screen shot 2017-10-10 at 2 16 00 pm

File: examples/amp-story/grid-layer-templates.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    <meta name="description" content="Get started with amp-story">
+
+    <meta name="viewport"
+          content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link rel="canonical" href="grid-layer-templates.html">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      body {
+        font-family: 'Roboto', sans-serif;
+      }
```
Comment (sebastianbenz) [inline examples/amp-story/grid-layer-templates.html]: suggestion, add:

```
amp-story-page {
  background: white;
}
```

File: examples/amp-story/animations-presets.html (hunk 3 above)
Comment (newmuis) [reply examples/amp-story/animations-presets.html]: Fixed by #11581

File: examples/amp-story/animations-presets.html (hunk 2 above)
Comment (newmuis) [reply examples/amp-story/animations-presets.html]: They're supposed to, but I think it's currently a bit buggy.

File: examples/amp-story/grid-layer-templates.html (hunk 4 above)
Comment (newmuis) [reply examples/amp-story/grid-layer-templates.html]: Done

File: examples/amp-story/helloworld.html (hunk 1 above)
Comment (newmuis) [reply examples/amp-story/helloworld.html]: Done

**Comment by newmuis** [comment]
/to @alanorozco can you help me merge this?

**Pull request merged**